app/components/Tab/Tab.tsx

```diff
@@ -1,4 +1,4 @@
-import { remote } from 'electron';
```

Comment:
Josh, ignore the changes in the file as they are not important to understanding this draft. I didn't mean to commit these changes. Was an experiment.